test/app7/binding.gyp

```diff
@@ -0,0 +1,13 @@
+{
+  "targets": [
+    {
+      "target_name": "<(module_name)",
+      "sources": [ "<(module_name).cc" ],
+      'product_dir': '<(module_path)',
+      "xcode_settings": {
+        "MACOSX_DEPLOYMENT_TARGET":"10.9",
```

Comment:
Just curious, is there any particular reason to set this?

Reply:
I created `app7` by copying one of the existing test apps as a starting point. The declaration came along for the ride. 

I appreciate your taking the time to review my code and for your constructive suggestions. Thank you.